android/app/src/main/java/com/mattermost/rnbeta/CustomPushNotification.java

```diff
@@ -265,8 +265,8 @@ protected void postNotification(int id, Notification notification) {
 
         boolean vibrate = notificationPreferences.getShouldVibrate();
         if (vibrate) {
-            // Each element then alternates between delay, vibrate, sleep, vibrate, sleep
-            notification.setVibrate(new long[] {1000, 1000, 500, 1000, 500});
+            // use the system default for vibration
+            notification.setDefaults(Notification.DEFAULT_VIBRATE);
```

Comment:
Will this affect the light and sound preferences?

Reply:
According to the documentation, this means that only the vibration settings should be inherited from the system defaults:
[Notification.Builder.html#setDefaults(int)](https://developer.android.com/reference/android/app/Notification.Builder.html#setDefaults(int))
"Set which notification properties will be inherited from system defaults."
The settings for light and sound are set explicitly and should therefore (in my opinion) not be touched by that.

Actually in my opinion we could also just use the default settings for sound and lights (building the bitmap depending on the settings instead of calling setLights and setSound and using setDefault once), but I wasn't sure if there's an intention for the custom light / sound.

Reply:
We do have settings that let you change the sound and lights so default for them are not an option

Reply:
I’ll be testing this tomorrow to see if the vibrate default setting works as expected.

Thanks so much for this, I’ll let you know if it works